app/helpers/student_teams_helper.rb

```diff
@@ -1,2 +1,16 @@
 module StudentTeamsHelper
+  # This method will return true if the assignment have some topic
+  def topic?(assignment_id)
+    topics = SignUpTopic.where("assignment_id = ?", assignment_id)
+    topics.empty?
+  end
+
+  def user_has_topic?(user_id, assignment_id)
```

Comment:
It will better to put this method in `team.rb`. Since topics will be assigned to teams.